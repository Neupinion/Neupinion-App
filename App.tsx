import { StatusBar } from 'expo-status-bar';
import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import CategoryCard from "./src/features/remakeissue/components/CategoryCard";

export default function App(): JSX.Element {
  return (
    <View style={styles.container}>
      <View>
        <CategoryCard></CategoryCard>
      </View>
      <StatusBar style="auto" />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: 'black',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
