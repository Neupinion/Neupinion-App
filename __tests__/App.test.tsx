import React from "react";
import renderer from 'react-test-renderer';

import App from "../App";

it('has 1 child', () => {
  const tree = renderer.create(<App/>).toJSON();
  expect(tree).toMatchSnapshot();
});
