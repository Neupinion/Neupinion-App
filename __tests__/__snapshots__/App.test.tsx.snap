// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has 1 child 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Open up App.tsx to start working on your app!
  </Text>
</View>
`;
